vendor/templates/vendor/manage/invoice_list.html

```diff
@@ -3,49 +3,49 @@
 
 {% block vendor_content %}
 <div class="container-fluid">
-    <nav aria-label="breadcrumb">
-        <!-- <ol class="breadcrumb"> -->
-        <h1>{% trans 'Site Order History' %}</h1>
-        <!-- </ol> -->
-    </nav>
-    <div class="row">
-        <div class="col">
-            <table class="table table-striped">
 
-                <thead>
-                    <tr>
-                        <th scope="col">{% trans '#' %}</th>
-                        <th scope="col">{% trans 'Name' %}</th>
-                        <th scope="col">{% trans 'State' %}</th>
-                        <th scope="col">{% trans 'Date' %}</th>
-                        <th scope="col">{% trans 'Currency' %}</th>
-                        <th scope="col">{% trans 'Total' %}</th>
-                    </tr>
-                </thead>
+  <nav aria-label="breadcrumb">
+    <!-- <ol class="breadcrumb"> -->
+    <h1>{% trans 'Site Order History' %}</h1>
+    <!-- </ol> -->
+  </nav>
+  <div class="row">
+    <div class="col">
+      <table class="table table-striped">
 
-                <tbody>
-                    {% for item in object_list %}
-                    <tr>
-                        <td><a class="text-primary"
-                                href="{% url 'vendor_admin:manager-order-detail' item.uuid %}">{{ item.pk }}</a></td>
-                        <td>{{ item }}</td>
-                        <td>{{ item.get_status_display }}</td>
-                        <td>{{ item.created }}</td>
-                        <td>{{ item.get_currency_display }}</td>
-                        <td>{{ item.total|default:"0.00" }}</td>
-                    </tr>
-                    {% empty %}
-                    <tr>
-                        <td>
-                            {% trans 'No Orders' %}
-                        </td>
-                    </tr>
-                    {% endfor %}
-                </tbody>
+        <thead>
+          <tr>
+            <th scope="col">{% trans '#' %}</th>
+            <th scope="col">{% trans 'Name' %}</th>
+            <th scope="col">{% trans 'Status' %}</th>
+            <th scope="col">{% trans 'Date' %}</th>
+            <th scope="col">{% trans 'Currency' %}</th>
+            <th scope="col">{% trans 'Total' %}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for item in object_list %}
+          <tr>
+            <td><a class="text-primary" href="{% url 'vendor_admin:manager-order-detail' item.uuid %}">{{ item.pk }}</a>
+            </td>
+            <td>{{ item.get_invoice_dispaly }}</td>
+            <td>{{ item.get_status_display }}</td>
+            <td>{{ item.created }}</td>
+            <td>{{ item.get_currency_display }}</td>
+            <td>$ {{ item.total|floatformat:2|default:"0.00" }}</td>
```

Comment:
see other notes on this same issue.  it could also be put into another function called `get_total_display()` if defaulting to 0.0 is an issue.  That method could be used to format the currency correctly.

Default filters in templates are not a good idea and are really intended for cases where you don't have control over the data returned.

Reply:
added the get_total_display function and replaced all default which were needed